app/models/question.rb

```diff
@@ -0,0 +1,26 @@
+class Question < ActiveRecord::Base
+  belongs_to :user
+  has_many :answers
+  has_one :best_answer, class_name: 'Answer'
```

Comment:
how does `best_answer` get determined? all this does right now is return the first answer.